### PR TITLE
Create REFERENCE.md

### PR DESCRIPTION
**Pull request by mikeshwe**
borrowing relevant info from https://github.com/astronomer/astro-sdk/blob/main/docs/OLD_README.md

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -0,0 +1,398 @@
...
+
+### The Table class
+
+To instantiate a table or bring in a table from a database into the `astro-sdk-python` ecosystem, you can pass a `Table` object into the class. This `Table` object will contain all of the metadata that's necessary for handling table creation between tasks. After you define a Table's metadata in the beginning of your pipeline, `astro-sdk-python` can automatically pass that metadata along to downstream tasks.
+
+In the following example, we define our table in the DAG instantiation. In each subsequent task, we pass in only an input `Table` argument because `astro-sdk-python` automatically passes in the additional context from our original `input_table` parameter.
+
+```python
+from astro-sdk-python import sql as aql
+from astro-sdk-python.sql.table import Table
```
Comment (kaxil) [inline REFERENCE.md:82]:
```suggestion
from astro import sql as aql
from astro.sql.table import Table
```
```suggestion
from astro-sdk-python import sql as aql
from astro-sdk-python.sql.table import Table
```

Comment (kaxil) [inline REFERENCE.md:82]:
```suggestion
from astro import sql as aql
from astro.sql.table import Table
```
```suggestion
from astro-sdk-python import sql as aql
from astro-sdk-python.sql.table import Table
```

File: REFERENCE.md [hunk 2]
```diff
@@ -0,0 +1,398 @@
...
+If you want to ensure that the output of your task is a cattle, you can declare it as a nameless `TempTable`. This places the output into your temp schema,
+which can be later bulk deleted. By default, all `aql.transform` functions will output to TempTables unless a `Table` object is used in the `output_table`
+argument.
+
+In the following example DAG, we set an `output_table` to a nameless `TempTable` meaning that any output from this DAG will be deleted once the DAG completes. If we wanted to keep our output, we would simply update the parameter to instantiate a `Table` instead.
+
+
+```python
+from astro-sdk-python import sql as aql
+from astro-sdk-python.sql.table import Table, TempTable
```
Comment (kaxil) [inline REFERENCE.md:115]:
```suggestion
from astro import sql as aql
from astro.sql.table import Table, TempTable
```
```suggestion
from astro-sdk-python import sql as aql
from astro-sdk-python.sql.table import Table, TempTable
```

Comment (kaxil) [inline REFERENCE.md:115]:
```suggestion
from astro import sql as aql
from astro.sql.table import Table, TempTable
```
```suggestion
from astro-sdk-python import sql as aql
from astro-sdk-python.sql.table import Table, TempTable
```

File: REFERENCE.md [hunk 3]
```diff
@@ -0,0 +1,398 @@
...
+## Loading Data
+
+To create an ELT pipeline, users can first load CSV or parquet data from either local, S3, or GCS into a SQL database with the `load_sql` function.
+To interact with S3, you must set an S3 Airflow connection in the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY`  environment variables.
+
+In the following example, we load data from S3 by specifying the path and connection ID for our S3 database in `aql.load_file`:
+
+```python
+from astro-sdk-python import sql as aql
+from astro-sdk-python.sql.table import Table
```
Comment (kaxil) [inline REFERENCE.md:145]:
```suggestion
from astro import sql as aql
from astro.sql.table import Table, TempTable
```
```suggestion
from astro-sdk-python import sql as aql
from astro-sdk-python.sql.table import Table
```

**Comment by tatiana** [comment]
Changes merged as part of https://github.com/astronomer/astro-sdk/pull/407

**Pull request closed without merging**